package.json

```diff
@@ -17,6 +17,9 @@
   "packageManager": "pnpm@9.7.1",
   "devDependencies": {
     "@fastify/pre-commit": "^2.1.0",
+    "@platformatic/next": "workspace:*",
```

Comment:
why in the root package json?

Reply:
Basically `pnpm` doesn't creates a `node_modules` folder in `packages/next/test/fixtures/next/standalone` or similar folders even if I create a `package.json` file.
So, since the Node resolver tries every `node_modules` folder up to `/`, I tricked it by adding the packages a `devDependency` of the root.

Reply:
why not add it as a dev dependency into the packages/next?

Reply:
It won't work. The `await import` is launched within `@platformatic/basic`. Adding it there would create a circular dependency.

Reply:
I understand why do you need them. But I'm not sure I like this hack. If @mcollina is good this it, I'm ok to go.

Reply:
@ivan-tymoshenko Good hint. I was able to solve it like that.